.changeset/red-poems-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Improves the asset caching by allowing stale assets to be revalidated [using entity tags](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) or the [Last-Modified](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified) date.
```

Comment:
```suggestion
Improves asset caching of remote images 

Astro will now store [entity tags](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) and the [Last-Modified](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified) date for cached remote images and use them to revalidate the cache when it goes stale.
```

- the PR description said this was caching of *remote* images, and if so, we should probably include that here!

- "by using this or that" could make it sound like the user has a choice (so they will be trying to figure out how to set it), or make them wonder which one will be used in their case. My suggestion above is sample wording to make it clear that this is NOT user configured behavior and instead is an explanation of what happens under the hood. Please correct as necessary!

- Re: other docs -- Since this is not user-configurable and does not allow/require any action from the user, then we *could* choose to simple explain e.g. how Astro determines whether cached items are stale or need revalidation, but we don't currently have documentation about this and it's not necessary to have for *this* PR. We have on our wish list more "technically, this is how this thing works" documentation as a nice-to-have, and this would count as one of those things! But this can be part of a larger effort at a later time as this does not interfere with someone *using* Astro.

Reply:
Wording changes look good, thanks!

As for changes to the docs I can create a PR to add a section for remote image caching. The reason I made this PR stemmed from not understanding why my images were not cached (server wasn't returning max-age), so documentation for this would be valuable.